### PR TITLE
Lints on the fly

### PR DESCRIPTION
**Pull request by skyrpex**
PHPCS actually can read from STDIN :P

### CODE REVIEW

**Comment by skyrpex** [comment]
Solves #52

**Comment by Arcanemagus** [comment]
Tested and working on Windows.

File: lib/main.coffee [hunk 1]
```diff
@@ -67,16 +67,16 @@ module.exports =
...
+        return helpers.exec(command, parameters, {stdin: text}).then (result) ->
           try
             result = JSON.parse(result.toString().trim())
           catch error
             atom.notifications.addError("Error parsing PHPCS response", {detail: "Check your console for more info. It's a known bug on OSX. See https://github.com/AtomLinter/Linter/issues/726", dismissable: true})
             console.log("PHPCS Response", result)
             return []
-          return [] unless result.files[filePath]
-          return result.files[filePath].messages.map (message) ->
+          return [] unless result.files['STDIN']
```
Comment (skyrpex) [inline lib/main.coffee]: I just noticed that I can remove the unnecesary [] accessor and the quotes...

**Comment by Arcanemagus** [comment]
No further comments have popped up, and as I'm apparently now a Collaborator here, merging :stuck_out_tongue:.

**Pull request merged**